.travis/install.sh

```diff
@@ -8,26 +8,31 @@ pushd common/ && pip install -e . && popd
 pushd pulpcore/ && pip install -e . && popd
 pushd plugin/ && pip install -e .  && popd
 
-export COMMIT_MSG=$(git log --format=%B -n 1 HEAD)
+export COMMIT_MSG=$(git show HEAD^2 -s)
 export PULP_FILE_PR_NUMBER=$(echo $COMMIT_MSG | grep -oP 'Required\ PR:\ https\:\/\/github\.com\/pulp\/pulp_file\/pull\/(\d+)' | awk -F'/' '{print $7}')
 export PULP_SMASH_PR_NUMBER=$(echo $COMMIT_MSG | grep -oP 'Required\ PR:\ https\:\/\/github\.com\/PulpQE\/pulp-smash\/pull\/(\d+)' | awk -F'/' '{print $7}')
-echo $COMMIT_MSG
-echo $PULP_FILE_PR_NUMBER
-echo $PULP_SMASH_PR_NUMBER
+
 if [ -z $PULP_FILE_PR_NUMBER ]; then
-  export PULP_FILE_SHA='master';
+  pip install git+https://github.com/pulp/pulp_file.git#egg=pulp_file
 else
-  export PULP_FILE_SHA=$(http https://api.github.com/repos/pulp/pulp_file/pulls/$PULP_FILE_PR_NUMBER | jq -r '.merge_commit_sha')
+  export PULP_FILE_SHA=$(curl https://api.github.com/repos/pulp/pulp_file/pulls/$PULP_FILE_PR_NUMBER | jq -r '.merge_commit_sha')
+  cd ../
+  git clone https://github.com/pulp/pulp_file.git
+  cd pulp_file
+  git fetch origin +refs/pull/$PULP_FILE_PR_NUMBER/merge
+  git checkout $PULP_FILE_SHA
+  pip install -e .
 fi
 
 if [ -z $PULP_SMASH_PR_NUMBER ]; then
-  PULP_SMASH_SHA='master';
+  pip install git+https://github.com/PulpQE/pulp-smash.git#egg=pulp-smash
 else
   export PULP_SMASH_SHA=$(http https://api.github.com/repos/PulpQE/pulp-smash/pulls/$PULP_SMASH_PR_NUMBER | jq -r '.merge_commit_sha')
+  cd ../
+  git clone https://github.com/PulpQE/pulp-smash.git
+  cd pulp-smash
+  git fetch origin +refs/pull/$PULP_SMASH_PR_NUMBER/merge
+  git checkout $PULP_SMASH_SHA
+  pip install -e .
```

Comment:
Wondering if we should DRY this up with a bash function?

Reply:
let's DRY up in a separate PR. please?